src/components v2/Atoms/Forms/InputDate/InputDate.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div class="ph-input-date-wrapper">
+    <p2-input-text
```

Comment:
**documentation:** Can we create a ticket for custom date formats?

I don't want to blockt he ticket for this but if we want to support future global endeavours we need to enable MM DD YYYY for the yanks 🇺🇸 🦅 🔫 🆓 

Can you make a ticket for it on the board so we dont forget about it?

Reply:
Yes, sounds good I'll create a ticket